.eslintrc.json

```diff
@@ -22,6 +22,9 @@
     "react/react-in-jsx-scope": "off",
     "react/jsx-uses-react": "off",
     "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx", ".ts", ".tsx"] }],
+    "@typescript-eslint/no-non-null-assertion": "off",
+    "@typescript-eslint/strict-boolean-expressions": "off",
+    "@typescript-eslint/no-floating-promises": "off",
```

Comment:
These are all good rules and imo shouldn't be turned off.
Another thing worth noting is that during this year our company has shifted quite a lot from ESLint towards Rome. https://rome.tools. Rome is a new kid on the block with auto-formatter. I'd highly recommend replacing ESLint in this project with Rome aswell.

Reference for that can be found in;
https://github.com/Metatavu/oss-surveys-management/blob/develop/.vscode/settings.json
https://github.com/Metatavu/oss-surveys-management/blob/develop/.vscode/extensions.json
https://github.com/Metatavu/oss-surveys-management/blob/develop/rome.json
https://github.com/Metatavu/oss-surveys-management/blob/25687220b496d089e317d7082bc2661ce29fbfc8/package.json#L48

Reply:
I would lean towards changing to Rome as well, even though it's my action that ESLint is used right now. We should discuss about this and maybe add a new task.

Reply:
Replaced ESLint with Rome.